src/Servers/Kestrel/Core/src/Http3Limits.cs

```diff
@@ -37,7 +39,7 @@ internal int HeaderTableSize
         /// <summary>
         /// Indicates the size of the maximum allowed size of a request header field sequence. This limit applies to both name and value sequences in their compressed and uncompressed representations.
         /// <para>
-        /// Value must be greater than 0, defaults to 8192
+        /// Value must be greater than 0, defaults to 2^14 (16,384).
```

Comment:
Changed `Http3Limits.MaxRequestHeaderFieldSize` to be consistent with `Http2Limits.MaxRequestHeaderFieldSize`.